test/io/test_io.py

```diff
@@ -883,6 +884,17 @@ def test_admin_live_dependent_on_main_app(defaultenv):
         response = postgrest.admin.get("/live")
         assert response.status_code == 503
 
+def test_admin_works_with_host_special_values(defaultenv):
+    "Should get a success from the admin live and ready endpoints when using special host values for the main app"
+
+    for host in ["*4", "!4", "*6", "!6", "*"]:
```

Comment:
This is not how multiple almost identical test-cases are created in pytest. Have a look at some of the other test cases in this file that use `@pytest.mark.parametrize` to achieve it in a better way.